CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 - `ListBox` no longer recreates itself after the list was updated
+- TypeScript update to 3.4.5
```

Comment:
This should be under `## [develop]` not under `## [3.5.0]`. Feels like a wrong base branch 🤔

Reply:
Base branch was right but I didn't look at the header :)